docs/UserGuide.md

```diff
@@ -251,6 +255,7 @@ Format: `weight INDEX [WEIGHT] `
 
 * Adds a weight value to a client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
 * Weight values specified in this command will be added as a new weight value to the specified client. To edit the latest weight of the client, use the [`edit`](#editing-a-client--edit) command.
+* If more than one value is entered, only the first value will be parsed. Extraneous parameters after the first value will be ignored.
```

Comment:
Might want to provide an example here for clarity.

Reply:
e.g. (weight 1 90 80 90 100)